lib/excision_web/live/classifier_live/index.html.heex

```diff
@@ -16,7 +16,12 @@
     end
   }
 >
-  <:col :let={{_id, classifier}} label="Name"><%= classifier.name %></:col>
+  <:col :let={{_id, classifier}} label="Name">
+    <%= classifier.name %>
+    <%= if Excision.Excisions.is_default_classifier?(classifier) do %>
+      <ExcisionWeb.Components.TooltipComponent.tooltip text='When a Decision Site is created, an associated "Passthrough" entity is also automatically created, which replicates the same behavior as making a function call to a model provider, with the added benefit that decisions coming from the LLM are recorded in your DecisionBox database.   Those decisions can now be inspected and labeled, establishing a baseline for accuracy before you replace it with a newly created, purpose-built classifier which can be trained through labeling for even greater accuracy.  Note that a Passthrough cannot be trained.' />
```

Comment:
there are some terms in here I am unclear about: What is a "Passthrough" entity? and what do we mean by function call? Function is a technical term, which may refer to certain kinds of technology created by Langchain or maybe an Azure function, or is it a function in a colloquial programming sense?

Also, I think this is too long to put in a tooltip. Judging by the fact that I am a little confused over the terminology, I think the best way forward is to either rephrase this so that it's ~1 sentence or maintain a docs page which the tooltip would link to.

It seems like there may be a lot of context to communicate here. I would make sure the docs really break everything down, with diagrams, examples, etc.

Reply:
CC @jwyss-allday

Reply:
thanks for the comments!
I think this breaks down to:
1. what is function calling - we hope they know this, but in the next doc update we can point to https://platform.openai.com/docs/assistants/tools/function-calling 
2. Even after reading this longish tooltip, it's still not clear what a "passthrough" entity is, is that correct?
3. The tooltip wording is a bit long to be a tooltip.
UX feedback has been that the 'baseline' classifier isn't really a classifier and it's really confusing why it's there.  So renaming it passthrough is an attempt to clarify that it's a special entity that will begin to instrument generative responses but still pass them through to the LLM provider. 
I'm not sure we've gotten this exactly right yet, but maybe this is an incremental improvement, and the next UX session can test and further refine it ?